concepticondata/conceptlists.tsv

```diff
@@ -214,3 +214,4 @@ Foley-1986-50	Foley, W. A.	1986		50	basic,areal	English	New Guinea		Foley1986	Fo
 Luniewska-2016-299	Łuniewska, Magdalena AND Haman, Ewa AND Armon-Lotem, Sharon AND Etenkowski, Bartłomiej AND Southwood, Frenette AND Anđelković, Darinka AND Blom, Elma AND Boerma, Tessel AND Chiat, Shula AND Engel de Abreu, Pascale AND Gagarina, Natalia AND Gavarró, Anna AND Håkansson, Gisela AND Hickey, Tina AND Jensen de López, Kristine AND Marinis, Theodoros AND Popović, Maša AND Thordardottir, Elin AND Blažienė, Agnė AND Cantú Sánchez, Myriam AND Dabašinskienė, Ineta AND Ege, Pınar AND Ehret, Inger-Anne AND Fritsche, Nelly-Ann AND Gatt, Daniela AND Janssen, Bibi AND Kambanaros, Maria AND Kapalková, Svetlana AND Kronqvist, Bjarke AND Kunnari, Sari AND Levorato, Chiara AND Nenonen, Olga AND Nic Fhlannchadha, Siobhán AND O’Toole, Ciara AND Polišenská, Kamila AND Pomiechowska, Barbara AND Ringblom, Natalia AND Rinker, Tanja AND Roch, Maja AND Savić, Maja AND Slančová, Daniela AND Tsimpli, Ianthi Maria AND Ünal-Logacev, Özlem	2016		299	acquisition	Afrikaans, Catalan, Danish, Dutch, English, Finnish, German, Greek, Hebrew, Hungarian, Icelandic, Irish, Xhosa, Italian, Lithuanian, Luxembourgish, Maltese, Polish, Russian, SouthAfricanEnglish, Serbian, Slovak, Spanish, Swedish, Turkish	Global	https://link.springer.com/article/10.3758%2Fs13428-015-0636-6	Luniewska2016		A list containing 299 items, listing the average age of acquisition for each respective item in 25 languages.	1154-1177	
 Gampe-2017-48	Gampe, Anja AND Kurthen, Ira AND Daum, Moritz M.	2017		48	acquisition	English	Global	http://journals.sagepub.com/doi/full/10.1177/0142723717736450	Gampe2017		A list that was developed to assess bilingual children's vocabulary (Swiss German native speakers). The MEAN* columns contain information about the average correctness in the lexical decision task for each item. Items are based on [Luniewska 299](:ref:Luniewska-2016-299).		
 Auracher-2017-16	Auracher, Jan	2017		16	acquisition	English	Global	https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5665516/	Auracher2017		Small list designed for eliciting reaction times to animal pictures. The animal list was included as one experiment (in a total of three different experiments) to differentiate between the reaction times of nonsense words and emotional body postures.		
+Pagel-2018-200	Pagel, Mark AND Meade, Andrew	2018		200	basic	English	Global	http://rstb.royalsocietypublishing.org/content/373/1740/20160517	Pagel2018		A list designed to serve as a basis for discussing the lexical replacement rates of number words. The list is based on [Blust 210](:ref:Blust-2008-210), [Dunn 207](:ref:Dunn-2012-207), and [Grollemund 100](:ref:Grollemund-2015-100). The RATE columns descrbibes the lexical replacement rate of an item per annum, i.e. its lexical stability. Note that the authors state that they took 102 meanings from [Grollemund 100](:ref:Grollemund-2015-100), whereas, to the best of our knowledge, the list only containts 100 meanings.		
```

Comment:
I suggest to reword:

```
The list is based on [Blust 210](:ref:Blust-2008-210), [Dunn 207](:ref:Dunn-2012-207), and [Grollemund 100](:ref:Grollemund-2015-100). The RATE columns descrbibes the lexical replacement rate of an item per annum, i.e. its lexical stability. Note that the authors state that they took 102 meanings from [Grollemund 100](:ref:Grollemund-2015-100), whereas, to the best of our knowledge, the list only containts 100 meanings.		
```
to 
```
The list is based on the [210-item list of ABVD](:ref:Blust-2008-210), the  [207-item list of IELex](:ref:Dunn-2012-207), and the [100-item list by Grollemund et al.](:ref:Grollemund-2015-100). The RATE column describes the lexical replacement rate of an item per year, i.e. its lexical stability. Note that the authors state that they took 102 meanings from [Grollemund et al. (2015)](:bib:Grollemund2015), whereas, to the best of our knowledge, the list only contains 100 meanings.	
```

Also, please provide the link to the SI, not the paper as URL.